main.go

```diff
@@ -137,6 +135,8 @@ func main() {
 		os.Exit(1)
 	}
 
+	kubeclient := kubernetes.NewForConfigOrDie(ctrl.GetConfigOrDie())
```

Comment:
nit, we already have kubeClient above, lets call this one kClient ? also its only used in L178 so might as well move this to L177 ? or just call `kubernetes.NewForConfigOrDie(ctrl.GetConfigOrDie())` directly in NewDrainReconcileController method